ui-lib/src/androidTest/java/co/electriccoin/zcash/ui/screen/scan/view/ScanPermissionGrantedViewTest.kt

```diff
@@ -45,6 +45,8 @@ class ScanPermissionGrantedViewTest : UiTestPrerequisites() {
     @Test
     @MediumTest
     fun check_all_ui_elements_displayed() {
+        composeTestRule.waitForIdle()
```

Comment:
This could long-term makes our tests flaky, as the waiting for the flow is technically idle.  Let's revisit when we add the additional integration tests.

Reply:
We've usually tried to deal with this by having idle waiting bumped up to the Activity layer, so that the views don't have asynchronous behavior.  But in this case, we'll have to think about it because I think bumping up to the Activity won't be the right solution.